packages/ebayui-core/src/components/ebay-filter-input/index.marko

```diff
@@ -0,0 +1,49 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+import { validSizes } from "./constants";
+$ const {
+    class: inputClass,
+    placeholder: inputPlaceholder,
+    a11yClearButton,
+    size: inputSize,
+    ...htmlInput
+} = input;
+
+$ var size = inputSize && validSizes.includes(inputSize) ? inputSize : null;
+
+$ const placeholder = inputPlaceholder ?? "Filter";
+
+<span class=["filter-input",
+    size && `filter-input--${size}`,
+    inputClass]>
+    <ebay-textbox
+        fluid
+
+        key="input"
+        placeholder=placeholder
+        on-keydown("emit", "keydown")
+        on-keypress("emit", "keypress")
+        on-keyup("emit", "keyup")
+        on-change("emit", "change")
+        on-input-change("emit", "input")
+        on-focus("emit", "focus")
+        on-blur("emit", "blur")
```

Comment:
Should we pass `type="search"` by default?

Reply:
Yes will update